xml/System.Threading/WaitHandle.xml

```diff
@@ -45,7 +45,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Threading.WaitHandle> class encapsulates Win32 synchronization handles, and is used to represent all synchronization objects in the runtime that allow multiple wait operations. For a comparison of wait handles with other synchronization objects, see [Overview of Synchronization Primitives](~/docs/standard/threading/overview-of-synchronization-primitives.md).  
+ The <xref:System.Threading.WaitHandle> class encapsulates native operating system synchronization handle, and is used to represent all synchronization objects in the runtime that allow multiple wait operations. For a comparison of wait handles with other synchronization objects, see [Overview of Synchronization Primitives](~/docs/standard/threading/overview-of-synchronization-primitives.md).  
```

Comment:
@rpetrusha maybe I've forgotten the "a" article. Is there a difference between
> The WaitHandle class encapsulates a native ... handle

and
> The WaitHandle class encapsulates native ... handles

If yes, then what is the difference? If not, I'm for the first variant, because using plural ("handles") makes me think that one `WaitHandle` instance might contain several OS handles.

Reply:
@pkulikov, I agree that "a synchronization handle" is preferable.  Sorry about that. Also, unrelated to your PR, the comma after "handle" should be removed.